lib/controllers/rocket.py

```diff
@@ -28,13 +29,13 @@ class RocketController():
     Enables:
        create a RocketpyRocket object from a Rocket model object.
     """
-    def __init__(self, rocket: Rocket):
+    def __init__(self, rocket: Rocket, rocket_option, motor_kind):
```

Comment:
What is `rocket_option` ?

Reply:
What values can this assume?

Reply:
It is the rocket to be selected from the list of rockets we have available under /data e.g: `power_off_drag=f"lib/data/{rocket_option.value.lower()}/powerOffDragCurve.csv`, for now the only available options are:

```
class RocketOptions(str, Enum):
    calisto: str = "Calisto"
    custom: str = "Custom"
```

And default is set to be "Calisto" as it is the reference rocket used in the getting started docs.

In the future, if Custom is selected, the API will expect the user to provide the powerOffDrag and powerOnDrag curves as arrays in the request.